assets/js/base/components/select/style.scss

```diff
@@ -118,3 +118,18 @@
 		display: none;
 	}
 }
+
+// This class is set on the body by Twenty Twenty-One when dark mode is active.
+.is-dark-theme {
```

Comment:
I didn't do a full-review, but just a quick note: when writing styles for a specific theme, we usually wrap them in a class, like so: `.theme-twentytwentyone { ... }`. You can take the [`CheckboxControl` styles](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/assets/js/base/components/checkbox-control/style.scss#L65) as a reference. :slightly_smiling_face:

Reply:
Thanks for pointing that out, I've updated the pull with this change now.